packages/pfff/pfff.0.37.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "yoann.padioleau@gmail.com"
+authors: [ "Yoann Padioleau <yoann.padioleau@gmail.com>" ]
+homepage: "https://github.com/facebook/pfff/wiki/Main"
+patches: [
+]
+build: [
+  ["./configure" "--novisual" "--nocmt" "--nobytecode"]
+  [make "depend"]
+  [make]
+  [make "opt"]
+]
+install: [
+  [make "install-libs"]
+]
+remove: [
```

Comment:
This part is unnecessary. Opam 2 already takes care of removing files that have been installed. Could you remove it?

Reply:
sure